mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -38,6 +37,17 @@ def listplaylist(context, name):
     return ['file: %s' % t.uri for t in playlist.tracks]
 
 
+def _lookup_playlist(context, name):
+    uri = context.lookup_playlist_uri_from_name(name)
+    playlist = uri is not None and context.core.playlists.lookup(uri).get()
+    if not playlist:
+        raise exceptions.MpdNoExistError('No such playlist')
+    track_uris = [track.uri for track in playlist.tracks]
+    tracks_map = context.core.library.lookup(uris=track_uris).get()
+    tracks = [track for u in track_uris for track in tracks_map[u]]
```

Comment:
Not a huge fan of these nested for loops like this, I tend to get confused about which one is the inner one.